.circleci/config.yml

```diff
@@ -87,6 +232,10 @@ jobs:
       - add_ssh_keys:
           fingerprints:
             - "a4:99:2e:fa:80:31:dc:23:25:7b:a5:70:3f:08:5f:fa"
+      - slack-basic:
+          event: always
+          branch_pattern: master
+          slack_mentions_user: << pipeline.parameters.slack_mentions_user >>
```

Comment:
Within `audius-protocol` I'm passing this around as a job parameter as well.

This seemed easier and perhaps a better style to call `pipeline.parameters` directly?

Reply:
Yeah, seems cleaner!